src/reports/controllers/reportsController.js

```diff
@@ -87,3 +89,40 @@ module.exports.postStats = async (req, res, next) => {
     }
     return res.status(204).json();
 };
+
+module.exports.getExportedReport = async(req, res, next) => {
+    let exportedReport;
+    try{
+        let reportInput;
+        try {
+            reportInput = await aggregateReportGenerator.createAggregateReport(req.params.test_id, req.params.report_id);
+        } catch (err) {
+            return next(err);
+        }
+        exportedReport = await reports.exportReport(reportInput,req.params.file_format);
+    } catch (err){
+        return next(err);
+    }
+    return res.send(exportedReport);
+}
+
+module.exports.getExportedCompareReport = async(req,res,next) => {
+    let exportedCompareReport;
+    try{
+        let aggregateReportArray =[];
+        let reportIds = req.query['reportIds'].split(",");
```

Comment:
1. we usually use query params with snake case so it can be report_ids and test_ids
2. I think once you go through the openapi spec and configure there it's an array, express framework will give you them as an array